stock_inventory_line_price/README.rst

```diff
@@ -0,0 +1,51 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================
+Standard price at inventory level
+=================================
+
+* This module allows you to handle the standard price at inventory level.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
```

Comment:
this part should be moved under https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage